docs/rules/prefer-numeric-literals.md

```diff
@@ -0,0 +1,41 @@
+# disallow `parseInt` in favor of binary, octal, and hexadecimal literals (prefer-numeric-literals)
+
+Binary and octal literals were introduced in ES6. Prior to this, `parseInt` was used to turn binary and octal strings into integers. This rule encourages use of binary, octal, and hexadecimal literals instead of `parseInt`.
+
+```js
+0b111110111 === 503;
+0o767 === 503;
+```
+
+## Rule Details
+
+This rule disallows `parseInt` if it is called with a radix option of 2 (binary), 8 (octal), or 16 (hexadecimal).
+
+Examples of **incorrect** code for this rule:
+
+```js
+/*eslint prefer-numeric-literals: "error"*/
+
+parseInt("111110111", 2) === 503;
+parseInt("767", 8) === 503;
+parseInt("1F7", 16) === 255;
+```
+
+Examples of **correct** code for this rule:
+
+```js
+/*eslint prefer-numeric-literals: "error"*/
+/*eslint-env es6*/
+
+parseInt(1);
+parseInt(1, 3);
+
+0b111110111 === 503;
+0o767 === 503;
+0x1F7 === 503;
+
+a[parseInt](1,2);
+
+parseInt(foo);
+parseInt(foo, 2);
+```
```

Comment:
In the past, the fact that rules are related to ES6-specific features seems to trump the fact that they are stylistic rules. I'm personally fine with leaving it in the ES6 category.